File: app/src/main/java/top/iwill/tinyapp/http/entity/LoginResult.java
package top.iwill.tinyapp.http.entity;

/**
 * Comment: //登录结果
 *
 * @author Jax.zhou in Make1
 * @date 2018/8/3
 * Company:Make1
 * Email:Jax.zhou@make1.cn
 */
public class LoginResult {

    /**
     * name :
     * token : 28c8edde3d61a0411511d3b1866f0636
     */

    private String name;

    private String token;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}

File: app/src/main/java/top/iwill/tinyapp/http/entity/LoginResult.kt
package cn.make1.cs.http.entity

import com.google.gson.annotations.SerializedName

/**
 * Comment: //登录结果实体解析类
 *
 * @author Jax.zhou in Make1
 * @date 2018/7/11
 * Company:Make1
 * Email:Jax.zhou@make1.cn
 */
class LoginResult {


    /**
     * uname : 15828420676
     * ulogo : /static/api/img_tmp/head.png
     * ugender : 1
     * token : 28c8edde3d61a0411511d3b1866f0636
     */

    @SerializedName("uname")
    var userName: String? = null

    @SerializedName("ulogo")
    var userHeadImg: String? = null

    @SerializedName("ugender")
    var userGender: Int = 1

    var token: String? = null
}
